api/src/main/java/org/geysermc/geyser/api/util/PlatformType.java

```diff
@@ -23,21 +23,17 @@
  * @link https://github.com/GeyserMC/Geyser
  */
 
-package org.geysermc.common;
+package org.geysermc.geyser.api.util;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-
-@Getter
-@AllArgsConstructor
-public enum PlatformType {
-    ANDROID("Android"),
-    BUNGEECORD("BungeeCord"),
-    FABRIC("Fabric"),
-    SPIGOT("Spigot"),
-    SPONGE("Sponge"),
-    STANDALONE("Standalone"),
-    VELOCITY("Velocity");
-
-    private final String platformName;
+/**
+ * Represents the platform Geyser is running on.
+ */
+public record PlatformType(String platformName) {
```

Comment:
Why is the old android platform here, but not the upcoming forge one?

Reply:
No reason to really remove it, and it may serve a purpose if it becomes feasible for us to support Android again. Forge support is a separate PR and will be added here separately.